lnbits/wallets/lndgrpc.py

```diff
@@ -80,9 +80,9 @@ def __init__(self):
         )
         encrypted_macaroon = settings.lnd_grpc_macaroon_encrypted
         if encrypted_macaroon:
-            macaroon = AESCipher(description="macaroon decryption").decrypt(
-                encrypted_macaroon
-            )
+            macaroon = AESCipher(
+                key=settings.auth_secret_key, description="macaroon decryption"
```

Comment:
Why is `settings.auth_secret_key` used here, this secret is related to the auth logic (for signing JWTs)?

Should a new `encryption_key` field be added for `LndRestFundingSource` & `LndGrpcFundingSource`

Reply:
If a key is not provided in the initialization of AESCipher here, then the call to decrypt will fail when the server runs as there is no user input in headless mode.  This is because the decrypt method in lnbits/lnbits/utils/crypto.py calls for passphrase, which checks if key is initialized or else prompts for it.

Any of the funding sources that use encryption would need to be updated. Should these use lnbits/helpers.py instead?